src/docfx/build/toc/TableOfContentsMapBuilder.cs

```diff
@@ -49,15 +48,26 @@ public TableOfContentsMap Build()
             // reverse the mapping between toc and documents
             // order by toc path
             var allTocs = new List<Document>();
+            var experimentalTocs = new List<Document>();
             foreach (var (toc, documents) in _tocToDocuments)
             {
                 if (_referencedTocs.ContainsKey(toc))
                 {
                     // referenced toc's mapping will be ignored
+                    // experimental toc will be ignored
                     continue;
                 }
 
-                allTocs.Add(toc);
```

Comment:
>allTocs.Add(toc); [](start = 16, length = 17)

no need to change

Reply:
what did i changed?

---
In reply to: [206506133](https://github.com/dotnet/docfx/pull/3127#discussion_r206506133) [](ancestors = 206506133)

Reply:
I mean no need to put in the `else` block...

---
In reply to: [206563535](https://github.com/dotnet/docfx/pull/3127#discussion_r206563535) [](ancestors = 206563535,206506133)